news/2020-02-18-mappers-report-february.md

```diff
@@ -0,0 +1,178 @@
+Another topic was if meetings should be held regularly in future. As long as there are topics which need discussion, the meetings will be held regularly. This should slow down in future, and when it does we will have more progress reports and fewer meetings. 
+
+### [Feb 15th, 2020 - Current BN Evaluations](https://osu.ppy.sh/community/forums/topics/1023943)
+
+The first point the meeting discussed was the activity threshold which is required for a Beatmap Nominator every month. The conclusion from the meeting is that Beatmap Nominators for osu!, osu!taiko and osu!catch need at least 3 nominations per month on average, and osu!mania will need 2 nominations on average each month. The activity will be regularly checked every two months by the NAT. Another point of the discussion was how nomination resets will affect the evaluation of the Beatmap Nominators. Unranks will no longer weigh more than regular nomination resets. 
+Last but not least, Beatmap Nominators will also be judged by the map quality of their nominations. This is to ensure a fair evaluation. Beatmap Nominators could have a high nomination count, but all maps could be of low quality. To ensure this does not happen the NAT will judge the quality of the nominations. 
+
+
+## <a id="withinbnnat"></a>Within the Beatmap Nominators and NAT
+
+### BN Additions
+
+Within the last few months we’ve had the chance to say hello to a lot of new members to the Beatmap Nominators, especially from the osu!taiko side of things! We’re very glad to have all these people on board whether they are new to the group or are returning veterans. We look forward to their help with pushing content to the Ranked section. As this is the first newspost, we will list all nominator additions for 2020 so far.
+
+**In osu!:** 
+[celerih](https://osu.ppy.sh/users/4696296), 
+[SMOKELIND](https://osu.ppy.sh/users/9327302) & 
+[Stack](https://osu.ppy.sh/users/6122935)
+
+**In osu!taiko:** 
+[\_DUSK\_](https://osu.ppy.sh/users/6092181), 
+[Capu](https://osu.ppy.sh/users/2474015), 
+[Gamelan4](https://osu.ppy.sh/users/9856910), 
+[Hoshikuz](https://osu.ppy.sh/users/9892196), 
+[incandescence](https://osu.ppy.sh/users/6256027), 
+[kanpakyin](https://osu.ppy.sh/users/394326), 
+[Stefan](https://osu.ppy.sh/users/626907), 
+[Vulkin](https://osu.ppy.sh/users/4901066) & 
+[Yales](https://osu.ppy.sh/users/2377881)
+
+**In osu!catch:** 
+[Lacrima](https://osu.ppy.sh/users/4915649) & 
+[Dapuluous](https://osu.ppy.sh/users/8140944)
+
+**In osu!mania:** 
+[Voxa](https://osu.ppy.sh/users/9229539), 
+[Dubstek](https://osu.ppy.sh/users/9555243), 
+[UnluckyCroco](https://osu.ppy.sh/users/9461160) & 
+[Amiichii](https://osu.ppy.sh/users/9228569)
+
+We’d like to use this section as a reminder that you can apply at any time over at https://bn.mappersguild.com/ as long as you reach the activity requirements. We hope to see some of your names in this list next month!
```

Comment:
```suggestion
We’d like to use this section as a reminder that you can apply as a Beatmap Nominator at any time over at [BN/NAT website](https://bn.mappersguild.com/) as long as you reach the activity requirements. We hope to see some of your names on this list next month!
```

Reply:
would add `the` before `BN/NAT website` as well